__tests__/app.test.js

```diff
@@ -3,34 +3,93 @@ const request = require("supertest");
 const connection = require("../db/connection");
 const seed = require("../db/seeds/seed");
 const data = require("../db/data/test-data");
+const endpoints = require("../endpoints.json")
 
 afterAll(() => {
-    return connection.end()
+  return connection.end();
 });
 
 beforeEach(() => {
-    return seed(data)
-})
+  return seed(data);
+});
 
-describe("GET api/topics", () => {
-    describe("Connecting to path", () => {
-        test("Should return status 200", () => {
-            return request(app).get("/api/topics").expect(200)
+describe("GET /api/topics", () => {
+  describe("Connecting to path", () => {
+    test("Should return status 200", () => {
+      return request(app).get("/api/topics").expect(200);
+    });
+    test("Should return the data from the path", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.length).toBe(data.topicData.length);
         });
-        test("Should return the data from the path", () => {
-            return request(app).get("/api/topics").expect(200)
-                .then(({ body }) => {
-                    expect(body.length).toBe(data.topicData.length)
-                });
+    });
+    test("Should return the correct data", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          body.forEach((topic) => {
+            expect(topic).toHaveProperty("slug", expect.any(String));
+            expect(topic).toHaveProperty("description", expect.any(String));
+          });
```

Comment:
Having this as a separate test is giving rise to a few repeated lines, this could be combined into the previous test and done altogether because it's utilising the same data